pkg/documentlog/store/sql.go

```diff
@@ -351,6 +377,50 @@ func (s sqlDocumentStore) migrate() error {
 	return nil
 }
 
+// updateContentHashes finds documents where contents_hash is missing (since it was added later on) and updates it where needed.
```

Comment:
mix of contentHash and contentsHash naming

Reply:
contents it is